snuba/processor.py

```diff
@@ -101,6 +101,8 @@ class ReplacementType(str, Enum):
         ReplacementType.TOMBSTONE_EVENTS,
         ReplacementType.EXCLUDE_GROUPS,
         ReplacementType.REPLACE_GROUP,
+        ReplacementType.START_UNMERGE_HIERARCHICAL,
+        ReplacementType.END_UNMERGE_HIERARCHICAL,
```

Comment:
Good catch.. though this is literally just all the members of `ReplacementType` enum... Do we actually need to define this here at all?